conda_build_config.yaml

```diff
@@ -78,10 +78,10 @@ g2clib:
   - 1.6
 gstreamer:
   - 1.14  # [not win]
-  - 1.18  # [win]
+  - 1.22  # [win]
```

Comment:
Do you think we could align the versions on all platforms or would you prefer to do that another day?

Reply:
I prefer another day.